ecs-cli/modules/cli/compose/entity/entity_helper.go

```diff
@@ -111,7 +116,7 @@ func GetOrCreateTaskDefinition(entity ProjectEntity) (*ecs.TaskDefinition, error
 	return resp, nil
 }
 
-func createRegisterTaskDefinitionRequest(taskDefinition *ecs.TaskDefinition) *ecs.RegisterTaskDefinitionInput {
+func createRegisterTaskDefinitionRequest(taskDefinition *ecs.TaskDefinition, tags []*ecs.Tag) *ecs.RegisterTaskDefinitionInput {
```

Comment:
To prevent posterity for having to go down the same rabbit hole I did, I make just make a small comment about how tags aren't a member on an ecs.TaskDefinition which is why we are passing them in this way.